packages/augur-test/src/tests/warp/Pruning.test.ts

```diff
@@ -1,38 +1,50 @@
 import { ContractInterfaces } from '@augurproject/core';
 import { SECONDS_IN_A_DAY } from '@augurproject/sdk';
-import { Seed, TestContractAPI } from '@augurproject/tools';
+import { Seed } from '@augurproject/tools';
 import { TestEthersProvider } from '@augurproject/tools/build/libs/TestEthersProvider';
 import { BigNumber } from 'bignumber.js';
 import { makeProvider } from '../../libs';
 import { ACCOUNTS, defaultSeedPath, loadSeed } from '@augurproject/tools';
-import Dexie from 'dexie';
+import * as IPFS from 'ipfs';
+import { tmpdir } from 'os';
+import { WarpTestContractApi } from '../../libs/warp-test-contract-api';
 
 describe('market pruning', () => {
+  let ipfs: Promise<IPFS>;
   let provider: TestEthersProvider;
-  let john: TestContractAPI;
-  let mary: TestContractAPI;
-  let mark: TestContractAPI;
+  let john: WarpTestContractApi;
+  let mary: WarpTestContractApi;
+  let mark: WarpTestContractApi;
   let seed: Seed;
 
   let market: ContractInterfaces.Market;
 
+  beforeAll(async () => {
+    ipfs = IPFS.create({
```

Comment:
@nuevoalex Did this pass locally for you? IPFS is a fickle party to involve in the tests.

Reply:
yup!